packages/expo-blur/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- `BlurView` can no longer be animated using the `Animated API`. Only `react-native-reanimated` is supported. ([#22783](https://github.com/expo/expo/pull/22783) by [@behenate](https://github.com/behenate))
```

Comment:
I wouldn't say this is a breaking change because it was not documented that you can animate the intensity. Instead, it's a new feature that we officially say that you can with Reanimated 😉